csrankings-d.csv

```diff
@@ -101,6 +101,7 @@ Dan C. Marinescu,University of Central Florida,https://www.cs.ucf.edu/~dcm,YoLdE
 Dan Chen 0001,Wuhan University,https://cs.whu.edu.cn/info/1019/2891.htm,wAzo3GUAAAAJ
 Dan E. Tamir,Texas State University,http://www.cs.txstate.edu/~dt19,zcdcbaIAAAAJ
 Dan E. Willard,University at Albany - SUNY,https://www.albany.edu/ceas/dan-willard.php,NOSCHOLARPAGE
+Dan F. DeBlasio,Carnegie Mellon University,https://www.dandeblasio.com/,3xE2SbUAAAAJ

```

Comment:
Only tenure-track faculty are eligible for inclusion.

Reply:
Removed as requested.